resources/event-sources/uninstall.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -eux -o pipefail
+resources=(
+  "httpsources.sources.kyma-project.io"
+)
+
+for resource in "${resources[@]}"; do
+  echo "starting deletion of ${resource} custom resources"
+  if kubectl get crd "${resource}"; then
+    kubectl \
+      delete \
+      --ignore-not-found \
+      "${resource}" \
+      --all-namespaces \
+      --all
+  else
+    echo "CRD ${resource} unknown to apiserver"
+  fi
+  echo "finished deletion of ${resource} custom resources"
+done
```

Comment:
(nit): new line

Reply:
I really would like to have a style guideline for that and also tooling around it. I personally don't care whether there is a newline or not. I would like to not have to wonder about it. I will leave it as it is until we have something like this :)